app/components/more-details.jsx

```diff
@@ -0,0 +1,125 @@
+  const classes = useStyles({ mode: 'light' })
+  const options = schema.enum || []
+  const label = uischema.label || schema.title
+
+  return (
+    <div>
+      <label>{label}</label>
+      <select value={data || ''} onChange={ev => handleChange(path, ev.target.value)} className={classes.select}>
+        <option value="" disabled>
+          Choose one
+        </option>
+        {options.map(option => (
+          <option key={option} value={option}>
+            {option}
+          </option>
+        ))}
+      </select>
+    </div>
+  )
+})
+
+const customRenderers = [...vanillaRenderers, { tester: rankWith(3, isEnumControl), renderer: CustomSelectRenderer }]
+
+const MoreDetails = props => {
+  const {
+    className = 'Submit Form',
+    schema = {},
+    uischema = {},
+    details = {},
+    onDone = () => {
+      valid, value
+    },
+    value,
+  } = props
+  const [data, setData] = useState(details)
+  const [isValid, setIsValid] = useState(false)
+  const classes = useStyles(props)
+
+  const handleOnClick = () => {
+    onDone({ valid: true, value })
+  }
+
+  const handleIsValid = () => {
+    const requiredData = ['householdIncome', 'housing', 'numberOfSiblings']
+    return requiredData.every(i => data[i])
+  }
+
+  useEffect(() => {
+    setIsValid(handleIsValid())
+  }, [data])
+
+  return (
+    <div className={cx(classes.moreDetailsContainer, className)}>
+      <p className={classes.title}>We just need a few more details about you to get started.</p>
+      <div className={classes.formContainer}>
+        <div>
+          <JsonForms
+            schema={schema}
+            uischema={uischema}
+            data={data}
+            renderers={customRenderers}
+            cells={vanillaCells}
+            onChange={({ data, _errors }) => setData(data)}
+          />
+        </div>
+        <PrimaryButton
+          primary
+          tile={'Submit form'}
+          className={classes.submitButton}
+          onClick={() => handleOnClick()}
+          onDone={onDone}
+          disabled={!isValid}
+        >
+          Submit
+        </PrimaryButton>
+      </div>
+    </div>
+  )
+}
+
+const useStyles = createUseStyles(theme => ({
+  moreDetailsContainer: props => ({
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    width: '100%',
+    backgroundColor: props.mode === 'dark' ? theme.colors.darkModeGray : theme.colors.white,
+    color: props.mode === 'dark' ? theme.colors.white : theme.colors.black,
+  }),
+  submitButton: {
+    width: '100%',
+    margin: '1rem 0',
+  },
+  title: props => ({
+    textAlign: 'center',
+    color: props.mode === 'dark' ? theme.colors.white : theme.colors.primaryButtonBlue,
+    fontSize: '2rem',
+  }),
+  formControl: {
+    marginBottom: '1rem',
+  },
+  select: {
+    width: '100%',
+    padding: '0.5rem',
+    borderRadius: '4px',
+    border: '0.1rem solid #EBEBEB',
```

Comment:
If these colors aren't in theme, please put them there and then use the colors out of theme.